libs/ojsonnet/Value_jsonnet.ml

```diff
@@ -1,74 +1,68 @@
-(* Yoann Padioleau
```

Comment:
Why removing this comment?
Please put it back, and you can add your name after mine at the top
(and change r2c in Semgrep Inc. instead)